kube/workspace/workspace-deployment.yaml

```diff
@@ -45,10 +45,7 @@ spec:
                 secretKeyRef:
                   name: huly-secret
                   key: SERVER_SECRET
-            - name: TRANSACTOR_URL
-              value: ws://transactor:3333;ws://localhost:3333
-            - name: NOTIFY_INBOX_ONLY
-              value: true
+              value: "true"
```

Comment:
value not needed?

Reply:
Did not notice that I left that 😅 , I've updated it.